src/env.h

```diff
@@ -426,6 +426,7 @@ constexpr size_t kFsStatsBufferLength =
 #define ENVIRONMENT_STRONG_PERSISTENT_VALUES(V)                                \
   V(async_hooks_after_function, v8::Function)                                  \
   V(async_hooks_before_function, v8::Function)                                 \
+  V(async_hooks_callback_trampoline, v8::Function)                             \
```

Comment:
can you remove the `domain_callback` a few lines below?

Reply:
Oh, good catch. Forgot to save that file before committing. 😅